workers/social/lib/social/models/group/index.coffee

```diff
@@ -385,6 +386,84 @@ module.exports = class JGroup extends Module
         callback null, memberData
 
 
+  # joinUser
+  #
+  # Joins user with given options to group either by logging in or converting
+  # them.
+  #
+  # @return {ResponseWithToken}
+  #
+  @joinUser = (options, callback) ->
```

Comment:
if you just add secure here, you will get the client, importing server/bongo is not required